THEY-Consulting/they-terraform
# switch azure function with storage trigger to v4

### PR instructions

setup:
```
cd examples/azure/function_app_with_storage_trigger
terraform init
terraform apply
```

test:
- go to azure portal to your function app and start log monitor
- upload a file to your storage account
- event should be visible in the logs

clenaup:
```
terraform destroy
```



### The steps of acceptance

✔ I executed the PR instructions and everything worked.

✔ I checked the requirements for the ticket, and they are matching the PR.

✔ I am satisfied with the code and left annotations if I had some ideas.
